stboot.go

```diff
@@ -31,21 +31,27 @@ import (
 	"github.com/u-root/u-root/pkg/uio"
 )
 
+const (
+	hostCfgDefault = "/etc/host_configuration.json"
+	hostCfgHelp = `Location of Host Configuration file.
+inside initramfs: "/path/to/host_configuration.json"
+STBOOT partition: "legacy:/path/to/host_configuration.json"
+as efivar:        "efivar:STDATA-d736a263-c838-4702-9df4-50134ad8a636"
```

Comment:
`STDATA-d736a263-c838-4702-9df4-50134ad8a636` STDATA ?
I guess it is an example for an ID, isn't it? 
What about `YOURID-d736a263-c838-4702-9df4-50134ad8a636`

Reply:
sounds good. done :+1: